aws-resources/src/main/java/io/opentelemetry/contrib/aws/resource/EcsResource.java

```diff
@@ -99,6 +99,24 @@ static void fetchMetadata(
     }
   }
 
+  @Nullable
+  private static String getAccountId(@Nullable String arn) {
+    if (arn != null) {
+      return arn.split(":")[4];
```

Comment:
avoid index out of bounds if data is invalid

Reply:
Done in d6dd717